arbitrum-docs/partials/_troubleshooting-users-partial.md

```diff
@@ -1,8 +1,57 @@
+### Why do I need ETH to use the Arbitrum network? {#why-do-i-need-eth-to-use-the-arbitrum-network}
+<p>When moving funds from Mainnet (L1) to Arbitrum (L2), you'll need ETH in your Arbitrum wallet. Even if you want only to use another, non-ETH, token, you will still need a small amount of ETH in your wallet on the Arbitrum network. This is because all Arbitrum transactions are powered by ETH. ETH is the currency used for gas fees.</p>
+
+<p></p>
+
+<p></p>
+
+
+
 ### Do I need to pay a tip / Priority fee for my Arbitrum transactions? {#do-i-need-to-pay-a-tip--priority-fee-for-my-arbitrum-transactions}
 <p>Since transactions are processed in the order that the Sequencer receives them, no priority fee is necessary for Arbitrum transactions; if a transaction does include a priority fee, it will be refunded to the transaction's origin address at the end of execution.</p>
```

Comment:
```suggestion
<p>Since transactions are processed in the order that the Sequencer receives them, no priority fee is necessary for Arbitrum transactions; if a transaction does include a priority fee, it will be refunded to the transaction's origin address at the end of the execution.</p>
```